viz.yaml

```diff
@@ -13,6 +13,9 @@ info:
       version: 0.3.8
       name: USGS-VIZLAB/vizlab
       ref: v0.3.8
+    bit64:
```

Comment:
what does this one do?

Reply:
For some reason, I couldn't read the tsv using fread without it. https://github.com/USGS-VIZLAB/gage-conditions/pull/16/commits/4e0880b83b16ad5202d4008fbd137eca60ccba2d